src/utils/hasProp.js

```diff
@@ -0,0 +1,9 @@
+function hasProp(obj, prop) {
+  if (!obj) return false;
+  if (Array.isArray(obj)) {
```

Comment:
Separate array item inclusion and property check?

Reply:
Agree, will add it along https://github.com/dashevo/wallet-lib/pull/295#discussion_r701729118